### PR TITLE
Add endpoint to get one event

### PR DESCRIPTION
**Pull request by barthy1**
[#136637059](https://www.pivotaltracker.com/story/show/136637059)

Signed-off-by: Konstantin Maksimov <kmaksimov@ru.ibm.com>

### CODE REVIEW

File: src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb [hunk 1]
```diff
@@ -6,6 +6,19 @@ module Api::Controllers
     class EventsController < BaseController
       EVENT_LIMIT = 200
 
+      get '/:id' do
+        content_type(:json)
+
+        event_id = params[:id]
+        event = Models::Event[event_id]
```
Comment (cppforlife) [inline src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb]: not sure if we need to turn it back into integer: https://github.com/cloudfoundry/bosh/blob/master/src/bosh-director/lib/bosh/director/api/event_manager.rb#L12 ?

File: src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb [hunk 2]
```diff
@@ -6,6 +6,19 @@ module Api::Controllers
...
+        if event.nil?
+          status(400)
```
Comment (cppforlife) [inline src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb]: 404?

File: src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb (hunk 1 above)
Comment (barthy1) [inline src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb]: @cppforlife it works as is. Do I need to update the code to support event ids as integer?

Comment (cppforlife) [inline src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb]: if it works i guess not. im somewhat surprised.

Comment (barthy1) [inline src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb]: I had a deeper look at it, and also don't understand the reason, why it works. So I've updated the code to feel more comfortable

**Pull request merged**